src/plugins/controls/public/constants.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const MIN_POPOVER_WIDTH = 300;
```

Comment:
I figured it would be ideal if every control popover had the same `panelMinWidth` for consistency - so, I made this a constant to be used in each implementation.

Reply:
💯